src/ad-services/live-connect/index.ts

```diff
@@ -78,7 +78,12 @@ export class LiveConnect extends BaseServiceSetup {
 		};
 	}
 
-	resolveAndReportId(idName: string, partnerName: string, params?: LiQParams) {
+	resolveAndReportId(idName: string, partnerName: string, defaultParams?: LiQParams) {
+		const params = {
+			...defaultParams,
+			qf: this.instantConfig.get<string>('icLiveConnectQf') || defaultParams.qf,
```

Comment:
```suggestion
			qf: this.instantConfig.get('icLiveConnectQf', defaultParams.qf),
```
this should properly infer type from `defaultParams.qf`